core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -87,12 +88,13 @@ __global__ __launch_bounds__(
   const DriverType &driver = *(reinterpret_cast<const DriverType *>(
       kokkos_impl_hip_constant_memory_buffer));
 
-  driver->operator()();
+  driver();
 }
 
 template <class DriverType>
 __global__ static void hip_parallel_launch_local_memory(
     const DriverType *driver) {
+  // FIXME_HIP driver() pass by copy
```

Comment:
Why is this a `TODO`?

Reply:
Because we cannot pass a driver by copy right now. This triggers a bug in the compiler.